drivers/spi/spi_dw.c

```diff
@@ -4,6 +4,12 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/*
+ * Copyright (c) 2023 Synopsys
```

Comment:
Why adding entire header here? Just add ` Copyright (c) 2023 Synopsys, Inc. All rights reserved.` in the 3-rd line above.